persist-cli/src/main/resources/version.properties

```diff
@@ -1 +1,3 @@
-version = ${project.version}
+#Wed Mar 08 22:18:01 IST 2023
+persistVersion=0.1.0
+version=${project.version}
```

Comment:
Do we need a project version here?

Reply:
This was already in that file. So, I am keeping it. But I couldn't find this usages. I will remove it.

Reply:
np. let's keep it for now